extentions.py

```diff
@@ -479,6 +479,12 @@ def register():
         precision=2,
         subtype='FACTOR'
     )
+    
+    Scene.bake_optimize_solid_materials = BoolProperty(
+        name="Optimize Solid Materials",
+        description="Optimizes solid materials by making a small area for them. AO pass will nullify",
+        default=True
```

Comment:
If you want this default on, I would add some extra logic to skip if AO or Normal passes are enabled.

Reply:
I did it for the material stuff, but I'll add it into the check for optimize materials just to be safe

Reply:
Also now I see why normal pass should disable it because baking normals also bakes into polygons righttt my bad